release/ray_release/anyscale_util.py

```diff
@@ -7,7 +7,7 @@
     from anyscale.sdk.anyscale_client.sdk import AnyscaleSDK
 
 
-LAST_LOGS_LENGTH = 10
+LAST_LOGS_LENGTH = 30
```

Comment:
what is this line change for? could you explain? and/or also add comment about how this var is used?

or, if it is used only once somewhere, move it to closer to the place that it is used?

Reply:
Ah I put a comment here but did not submit. This line is how many line of logs we want to capture from the log file. I need it to be 30 so that it's long enough to capture the useful stack trace.